step.yml

```diff
@@ -82,8 +82,8 @@ inputs:
       is_expand: true
       is_required: true
       value_options:
-        - true
-        - false
+        - "true"
```

Comment:
The new dev. the guideline says it should be "yes" or "no"
https://github.com/bitrise-io/step-development-guideline/blob/develop/inputs.md#step-input-values